R/te_datastore_csv.R

```diff
@@ -96,3 +96,49 @@ setMethod(
     data_table
   }
 )
+
+
+#' @rdname te_datastore_csv-class
+#' @include trial_sequence.R
+setMethod(
+  f = "sample_expanded_data",
+  signature = "te_datastore_csv",
+  definition = function(object, p_control, period, subset_condition = NULL, seed) {
+    old_seed <- globalenv()$.Random.seed
+    on.exit(suspendInterrupts(set_random_seed(old_seed)))
+    set.seed(seed)
+
+    all_periods <- NULL
+    for (n in seq_len(length(object@files))) {
```

Comment:
maybe this is neater
```suggestion
    for (n in seq_along(object@files)) {
```